src/encoded/static/components/objectutils.js

```diff
@@ -1178,3 +1178,26 @@ export const collectDatasetBiosamples = (dataset) => {
     }
     return [];
 };
+
+
+/**
+ * Display the current and maximum character counter for a <textarea>.
+ */
+export const TextAreaCounter = ({ value, maxLength, css }) => (
+    <div className={`textarea-counter${css && ` ${css}`}`}>
+        {value.length} / {maxLength}
```

Comment:
Do you need to Math. {floor/ceil} this so you always get a whole number?

Reply:
This is a literal slash; not division, e.g. “50 / 1000.”

Reply:
Opps. Thanks.